Remotion/ObjectBinding/Web/UI/Controls/BocValidationErrorIndicatorColumnDefinition.cs

```diff
@@ -39,5 +44,21 @@ protected override IBocColumnRenderer GetRendererInternal (IServiceLocator servi
 
     /// <inheritdoc />
     protected override string DisplayedTypeName => "ValidationErrorIndicatorColumnDefinition";
+
+    /// <summary> Gets or sets a flag that determines whether to enable sorting for this column. </summary>
+    [PersistenceMode(PersistenceMode.Attribute)]
+    [Category("Behavior")]
+    [Description("A flag determining whether to enable sorting for this column.")]
+    [DefaultValue(true)]
+    [NotifyParentProperty(true)]
+    public bool IsSortable { get; set; } = true;
+
+    public IComparer<BocListRow> CreateCellValueComparer ()
```

Comment:
Sorry, I just noticed: we're missing the tests here. please check BocSimpleColumnDefitionTest